src/controllers/news.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express');
+const router = express.Router();
+const path = require('path');
+const fetch = require('node-fetch');
+const { URLSearchParams } = require('url');
```

Comment:
I guess `URLSearchParams` is a global class and you can use it without requiring it